File: core/src/main/java/no/elg/infiniteBootleg/world/SinglePlayerWorld.java
package no.elg.infiniteBootleg.world;

import no.elg.infiniteBootleg.world.generator.ChunkGenerator;
import org.jetbrains.annotations.NotNull;

public class SinglePlayerWorld extends ClientWorld {

  public SinglePlayerWorld(
      @NotNull ChunkGenerator generator, long seed, @NotNull String worldName) {
    super(generator, seed, worldName);
  }
}

File: core/src/main/kotlin/no/elg/infiniteBootleg/world/world/SinglePlayerWorld.kt
package no.elg.infiniteBootleg.world.world

import no.elg.infiniteBootleg.world.generator.chunk.ChunkGenerator

class SinglePlayerWorld(generator: ChunkGenerator, seed: Long, worldName: String, forceTransient: Boolean = false) : ClientWorld(generator, seed, worldName, forceTransient)
